pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "tor"
+version = "3.6.0"
+description = "The leader of the show for r/TranscribersOfReddit!"
+authors = ["Grafeas Group Ltd. <devs@grafeas.org>"]
+license = "MIT"
+repository = "https://github.com/grafeasgroup/tor"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+sh = "^1.12"
+tor_core = {git = "https://github.com/grafeasgroup/tor_core"}
+pytest = {version = "^4.2",optional = true}
+pytest-cov = {version = "^2.6",optional = true}
+codacy-coverage = {version = "^1.3",optional = true}
```

Comment:
This probably shouldn't be part of the regular dependencies. We can manually pip install it on the Travis-CI run, since that's the only time we would need it installed.

Reply:
That defeats the purpose of having all the dependencies in one place, though 😛 
Either way, it will be only be installed when the extras `ci` package is installed.